src/dependencyReportPanel.ts

```diff
@@ -158,5 +163,21 @@ let render_project_failure = () => {
 
 let render_stack_iframe = portaluri => {
   //const result = sa.result[0];
-  return `<iframe width="100%" height="100%" frameborder="0" src=${portaluri} id="frame2" name="frame2"></iframe>`;
+  return `<iframe id="frame" width="100%" height="100%" frameborder="0" src=${portaluri} sandbox="allow-scripts allow-popups allow-popups-to-escape-sandbox allow-same-origin"></iframe>
```

Comment:
Why do you need sandbox attributes? It should work even without that.

Reply:
Ya, we don't need them any more, I think i forgot to delete them after testing them out.